pixart.js

```diff
@@ -1 +1,36 @@
+//add 2 more brushes
+$(".brush").attr("class", "brush 1");
+$(".controls").append("<div class='brush 2'></div>");
+$(".controls").append("<div class='brush 3'></div>");
 
+//change $(".brush").css("background-color") to $("#color-field").val() on click of $("#set-color")
+
+// $("#set-color").on("click", function(e){
+//   e.preventDefault();
+//   $(".brush").css("background-color", $("#color-field").val())
+// })
+
+var clickCount = 0;
+
+$("#set-color").on("click", function(e){
+  e.preventDefault();
+  $(".brush").eq(clickCount).css("background-color", $("#color-field").val());
+  if (clickCount < 2) {
```

Comment:
Great start on the bonus. Can you think of a way to do this so that rather than cycle through the swatches with each click, the new color is always placed in the first swatch (and that pushes the two next most recent colors to the 2nd and 3rd swatches)?